backend/src/chats/chats.gateway.ts

```diff
@@ -0,0 +1,51 @@
+import {
+	MessageBody,
+	OnGatewayConnection,
+	SubscribeMessage,
+	OnGatewayDisconnect,
+	OnGatewayInit,
+	WebSocketGateway,
+	WebSocketServer,
+	ConnectedSocket,
+} from '@nestjs/websockets';
+import { Server, Socket } from 'socket.io';
+import { SocketService } from '../socket/socket.service';
+import { ChatId } from '../socket/socket.types';
+
+@WebSocketGateway({
+	namespace: '/chats',
+	cors: {
```

Comment:
Cross-Origin Resource Sharing is an http-header?????

Reply:
yes